src/gtk/artgtk.cpp

```diff
@@ -58,7 +58,18 @@ namespace
     #define ART(wxId, stockId, unused) wxId, stockId,
```

Comment:
This should bechanged to  `wxArtStockMapping(wxId, stockId),` now if only to avoid `-Wmissing-braces` warnings (but also for clarity).